janitor/functions/encode_categorical.py

```diff
@@ -22,82 +22,80 @@ def encode_categorical(
 
     This method does not mutate the original DataFrame.
 
-    Note: In versions < 0.20.11, this method mutates the original DataFrame.
-
-    If `categories` is `None` in the `kwargs` tuple, then the
-    values for `categories` are inferred from the column;
-    if `order` is `None`, then the values for `categories` are applied unordered.
+    Simply pass a string, or a sequence of column names to `column_names`;
+    alternatively, you can pass kwargs, where the keys are the column names
+    and the values can either be None, or a string(either `sort` or `appearance`),
```

Comment:
```suggestion
    and the values can either be None, `sort`, `appearance`
```
is a little more concise.

I'm also thinking that we can clarify a little what each option means? (feel free to rephrase or integrate this info however you see fit;)

- None: column is cast to an unordered categorical
- 'sort': column is cast to an ordered categorical, with the order defined by the sort-order of the categories.
- 'appearance': column is cast to an ordered categorical, with the order defined by the order of appearance in the original column.
- a 1d-array-like object: column is cast to an ordered categorical, with the categories and order as specified in the input array.